src/scenes/_Router/GalleryRoute.tsx

```diff
@@ -86,7 +96,21 @@ export default function GalleryRoute({
     }
   }, [footer, footerVisibleOutOfView, footerVisibleWithinView, footerIsFixed, isMobile]);
 
-  const banner = useMemo(() => <Banner text="" queryRef={query} />, [query]);
+  const banner = useMemo(() => {
+    return isFeatureEnabled(FeatureFlag.POSTER_PAGE) ? (
+      <Banner
+        title={countdownTimer}
+        text="Thank you for being a member of Gallery. Celebrate our new brand with us by signing our 2022 Community Poster that we will mint as an NFT."
+        queryRef={query}
+        localStorageKey="gallery-poster-banner"
```

Comment:
tiny nit: let's use underscores for the local storage key `gallery_poster_banner` for consistency with other storage keys. the other keys are defined in `storageKeys.ts` so we can do the same for this too